@here/harp-mapview/lib/Watermark.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as THREE from "three";
+
+/**
+ * Creates and handles rendering of watermark logo geometry for [[MapView]].
+ */
+export class Watermark {
```

Comment:
The way it's rendererd - just texture in corner (no opacity) and no real watermarking of whole area of image area doesn't fit common definition of watermark in digital imaging.

Consider more precise documentation what it really does and possible change name completly.

Maybe